packages/@react-stately/virtualizer/src/useVirtualizerState.ts

```diff
@@ -77,6 +77,8 @@ export function useVirtualizerState<T extends object, V, W>(opts: VirtualizerPro
   // eslint-disable-next-line arrow-body-style
   useEffect(() => {
     return () => virtualizer.willUnmount();
+  // We only need to cleanup the raf on unmount hence the empty dep array
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
why not `};`?

Reply:
the dep array is required, if i understood what you're asking